src/Mono.Android/Android.Runtime/JavaObject.cs

```diff
@@ -92,8 +92,8 @@ public JavaObject (object inst)
 
 		protected override Java.Lang.Object Clone ()
 		{
-			if (inst is ICloneable c)
-				return new JavaObject (c.Clone ());
+			if (inst is ICloneable)
```

Comment:
Is this change *actually* necessary*?  It feels like the "main" part of the fix is in `Mono.Android.xml`, removing the `<type fullname="Android.Runtime.JavaObject"/>` entry.

How could `inst is ICloneable c` cause the linker to pull in things?  That doesn't make sense.

Reply:
It is not necessary anymore as we are removing the Clone method, hooray.

It was pulling things in, because `ICloneable:.Clone()` was preserved, and thus also `Clone` methods on  other preserved types implementing `ICloneable` were preserved as result.